standard/interfaces.md

```diff
@@ -216,58 +218,153 @@ The members of an interface are the members inherited from the base interfaces a
-- The name of a property or event shall differ from the names of all other members declared in the same interface.
-- The signature of an indexer shall differ from the signatures of all other indexers declared in the same interface.
+A member `M2` with definition in an interface requires any overriding member to declare an explicit interface implementation (§19.6.2). That member `M2` only be accessed through an interface instance.
 
-The inherited members of an interface are specifically not part of the declaration space of the interface. Thus, an interface is allowed to declare a member with the same name or signature as an inherited member. When this occurs, the derived interface member is said to *hide* the base interface member. Hiding an inherited member is not considered an error, but it does cause a compiler to issue a warning. To suppress the warning, the declaration of the derived interface member shall include a `new` modifier to indicate that the derived member is intended to hide the base member. This topic is discussed further in [§7.7.2.3](basic-concepts.md#7723-hiding-through-inheritance).
+Some declarations, such as *constant_declaration* (§15.4) have no restrictions in interfaces.
 
-If a `new` modifier is included in a declaration that doesn’t hide an inherited member, a warning is issued to that effect. This warning is suppressed by removing the `new` modifier.
+The inherited members of an interface are specifically not part of the declaration space of the interface. The rules for inheritance in classes (§15.3.4) do not apply to interfaces. The `new` modifier can hide an interface member declared in a base interface. The rules for the `new` modifier in classes (§15.3.5) apply to interface member declarations.
 
 > *Note*: The members in class `object` are not, strictly speaking, members of any interface ([§19.4](interfaces.md#194-interface-members)). However, the members in class `object` are available via member lookup in any interface type ([§12.5](expressions.md#125-member-lookup)). *end note*
 
 The set of members of an interface declared in multiple parts ([§15.2.7](classes.md#1527-partial-type-declarations)) is the union of the members declared in each part. The bodies of all parts of the interface declaration share the same declaration space ([§7.3](basic-concepts.md#73-declarations)), and the scope of each member ([§7.7](basic-concepts.md#77-scopes)) extends to the bodies of all the parts.
 
-### 19.4.2 Interface methods
+> *Example*: Consider an interface `IA` with an implementation for a member `M`. As `M` is not abstract, outside that interface or any interface derived from it, that name is not visible. It must be accessed through a reference whose compile-time type is an interface that is implicitly convertible to `IA`.
```

Comment:
@BillWagner to consider whether this example is useful or not. (The explanation is not helpful at the moment.)

If not, let's delete it - and the markdownlint-disable following it.